starlite/handlers/asgi.py

```diff
@@ -23,12 +23,12 @@ class ASGIRouteHandler(BaseRouteHandler["ASGIRouteHandler"]):
     @validate_arguments(config={"arbitrary_types_allowed": True})
     def __init__(
         self,
-        path: Union[Optional[str], Optional[List[str]]] = None,
+        path: Optional[Union[str, Sequence[str]]] = None,
```

Comment:
The types changed here also need to be changed in the docstring, but I can't comment on that